### PR TITLE
feature/GH-13048: move i18n to cart-lib

### LINKED ISSUE #13048: Move i18n to cart-lib
Move i18n to cart-lib and deprecate the old i18n labels

### PR DESCRIPTION
**Pull request by WeizhengSap**
close #13048

### CODE REVIEW

File: projects/assets/src/translations/en/cart.ts [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/**
+ * @deprecated since 4.2, use feature-library @spartacus/cart/main/assets instead
+ */
```
Comment (plabadie) [inline projects/assets/src/translations/en/cart.ts]: Why not remove the deprecated code instead? @spartacus/cart/main/assets is 5.0, not 4.2

File: projects/assets/src/translations/translation-chunks-config.ts [hunk 2]
```diff
@@ -16,6 +16,9 @@ export const translationChunksConfig: TranslationChunksConfig = {
     'formErrors',
     'errorHandlers',
   ],
+  /**
+   * @deprecated since 4.2, use feature-library @spartacus/cart/main/assets instead
+   */
```
Comment (plabadie) [inline projects/assets/src/translations/translation-chunks-config.ts]: Why not remove the deprecated code instead? @spartacus/cart/main/assets is 5.0, not 4.2

File: feature-libs/cart/main/assets/translations/en/cart.i18n.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  saveForLaterItems: {
+    itemTotal: 'Saved for later ({{count}} item)',
+    itemTotal_plural: 'Saved for later ({{count}} items)',
+    cartTitle: 'Cart',
+    saveForLater: 'Save For Later',
+    moveToCart: 'Move To Cart',
+    stock: 'Stock',
+    forceInStock: 'In Stock',
+  },
+};
```
Comment (plabadie) [inline feature-libs/cart/main/assets/translations/en/cart.i18n.ts:55]: Should there also be wishlist labels in this file?

File: projects/assets/src/translations/en/cart.ts (hunk 1 above)
Comment (WeizhengSap) [reply projects/assets/src/translations/en/cart.ts]: removed

File: feature-libs/cart/main/assets/translations/en/cart.i18n.ts (hunk 3 above)
Comment (WeizhengSap) [reply feature-libs/cart/main/assets/translations/en/cart.i18n.ts:55]: the `wishlist` chunk is inside 'my account`. The original cart chunks contain it as well. Let me check.

Comment (WeizhengSap) [reply feature-libs/cart/main/assets/translations/en/cart.i18n.ts:55]: I move it from `my account` to `cart`

**Review by plabadie — APPROVED** [review]

**Pull request merged**